### PR TITLE
Add fish shell install script to the script library

### PR DESCRIPTION
**Pull request by andreiborisov**
Resolves https://github.com/microsoft/vscode-dev-containers/issues/570

### CODE REVIEW

**Comment by Aisuko** [comment]
@andreiborisov  Please update the REAME.md to solve the conflict

File: script-library/fish-debian.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    if grep -q 'stretch' < /etc/os-release; then
+        echo 'deb http://download.opensuse.org/repositories/shells:/fish:/release:/3/Debian_9.0/ /' | tee /etc/apt/sources.list.d/shells:fish:release:3.list
+        curl -fsSL https://download.opensuse.org/repositories/shells:fish:release:3/Debian_9.0/Release.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/shells_fish_release_3.gpg > /dev/null
+    elif grep -q 'buster' < /etc/os-release; then
+        echo 'deb http://download.opensuse.org/repositories/shells:/fish:/release:/3/Debian_10/ /' | tee /etc/apt/sources.list.d/shells:fish:release:3.list
+        curl -fsSL https://download.opensuse.org/repositories/shells:fish:release:3/Debian_10/Release.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/shells_fish_release_3.gpg > /dev/null
+    fi
+    apt update
+    apt install -y fish
+fi
```
Comment (Aisuko) [inline script-library/fish-debian.sh:57]: I have a consider here, Should we add the `apt autoremove` and remove the cache here in order to optimize the image size?

**Review by Aisuko — CHANGES_REQUESTED** [review]
Please kindly check the comment

File: script-library/fish-debian.sh (hunk 1 above)
Comment (andreiborisov) [reply script-library/fish-debian.sh:57]: Indeed, I have nothing against it👍

Comment (Aisuko) [reply script-library/fish-debian.sh:59]: Thank you

**Comment by Aisuko** [comment]
@Chuxel Hello sir, please kindly take a look this feature

**Comment by 2percentsilk** [comment]
Hi there! Apologies for the delay - before we merge, I'd just like to verify that @andreiborisov is willing to maintain this script moving forward. This would entail helping us update the script if we're made aware of any vulnerabilities or consulting on PRs that other community members may open that affect this script.

**Comment by andreiborisov** [comment]
Yes, I commit to maintain this script going forward👍

**Comment by andreiborisov** [comment]
Any ETA on the merge?

**Pull request merged**